binaries/scheduler/main.go

```diff
@@ -7,35 +7,30 @@ import (
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/sched/config"
 	"github.com/scootdev/scoot/scootapi/server"
-	"golang.org/x/net/context"
 )
 
 var addr = flag.String("addr", "localhost:9090", "Bind address for api server.")
-var cfgFile = flag.String("cfg", "", "Path to sched overlay config file.")
+var cfgText = flag.String("sched_config", "", "Scheduler Configuration.")
 
 func main() {
 	log.Println("Starting Cloud Scoot API Server & Scheduler")
 	flag.Parse()
 
-	plugins := config.Plugins{}
-	ctx := context.Background()
-	transportFactory := thrift.NewTTransportFactory()
-	protocolFactory := thrift.NewTBinaryProtocolFactoryDefault()
+	parser := config.DefaultParser()
 
 	// Construct scootapi server handler based on config.
-	cfg, err := config.DefaultConfig(*cfgFile, plugins)
-	if err != nil {
-		log.Fatal("Error configuring Scoot API: ", err)
-	}
 
-	handler, err := config.ConfigureCloudScoot(cfg, ctx, transportFactory, protocolFactory)
+	handler, err := parser.Create([]byte(*cfgText))
```

Comment:
How are we planning to cleanly stop this if we don't pass a context? Are we punting on that?